generated/niscope/niscope.proto

```diff
@@ -1700,6 +1701,19 @@ message GetScalingCoefficientsResponse {
   sint32 number_of_coefficient_sets = 3;
 }
 
+message GetStartTimestampInformationRequest {
+  nidevice_grpc.Session vi = 1;
+}
+
+message GetStartTimestampInformationResponse {
+  int32 status = 1;
+  uint32 sys_time_in128_bits_t1 = 2;
```

Comment:
Is this supposed to be `sys_time_in_128_bits_t1`?

Reply:
Yeah, I'll need to add `grpc_name` for these fields in the metadata.

Reply:
Added `grpc_name` so these look normal now with `sys_time_in_128_bits_t<number>`